lib/llvm-backend/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 
 [dependencies]
 wasmer-runtime-core = { path = "../runtime-core", version = "0.5.4" }
-inkwell = { git = "https://github.com/wasmerio/inkwell", branch = "llvm7-0" }
+inkwell = { git = "https://github.com/lumen/inkwell", rev = "64921e418c3e8f4b32f9beed56ebd5aac4e677c7", features = ["llvm8-0"] }
```

Comment:
Can we publish this branch in our own inkwell repo?

Reply:
Done. https://github.com/wasmerio/inkwell/tree/llvm8-0